UNRELEASED.md

```diff
@@ -1 +1,6 @@
 # Unreleased
+
+**Added:**
+ - react-native-bpk-component-badge:
+     - Badges now accept an optional `accessoryView` that allows icons, or other content, to be placed alongside the text inside the bade. See https://backpack.github.io/components/badge/?platform=native#with-multiple-icons
```

Comment:
You changed the ID in the docs to `with-icons` so the link will need to change too.

Reply:
🤦‍♂️